hack/e2e.go

```diff
@@ -146,21 +146,21 @@ func main() {
 		}
 	}
 
-	failure := false
+	var success bool
```

Comment:
Maybe simply leaving everything as is and just doing `failure = !Test()` might be enough to fix this bug?